src/test/resources/rca/rca.conf

```diff
@@ -41,5 +41,7 @@
     // How often the sqlite file be repeated in seconds. This file contains RCAs and therefore rotating it too frequently
     // might not be as fruitful as there might not be any data.
     "rotation-period-seconds": 21600
-  }
+  },
+
+    "muted-rcas": "CPU_Utilization, Heap_AllocRate"
```

Comment:
Maybe, we should not add muted RCA nodes to these configs that are generically imported by all tests. This will mute some RCA nodes underneath. As muted RCA is a special config, we should exclusively create rca.conf for these test cases. Thoughts ?

Reply:
I was debating the need for having a separate config file for testing muted RCAs to ensure the orthogonality. Have added one.